scripts/publish-root.gradle

```
@@ -28,8 +28,8 @@ nexusPublishing {
             username = ossrhUsername
             password = ossrhPassword
             version = PUBLISH_VERSION
-            nexusUrl.set(uri("https://s01.oss.sonatype.org/service/local/"))
-            snapshotRepositoryUrl.set(uri("https://s01.oss.sonatype.org/content/repositories/snapshots/"))
+            nexusUrl.set(uri("https://ossrh-staging-api.central.sonatype.com/service/local/"))
+            snapshotRepositoryUrl.set(uri("https://central.sonatype.com/repository/maven-snapshots/"))
```

```suggestion
            snapshotRepositoryUrl = uri("https://central.sonatype.com/repository/maven-snapshots/")
```

we don't need to explicitly call `set` i suppose